poc/Makefile

```diff
@@ -1,6 +1,6 @@
 test:
-	sage -python mechanism/binary_randomized_response.py
```

Comment:
Rest of the changes are for reference implementation. Feel free to ignore @kunal-talwar .